### DISCUSSION TITLE
Wurde bei scripts was geändert? erhalte kein event.path mehr?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2198] [Q&A] [answered]
Beim "click" event wurde sonst immer ein event.path mitgeliefert.
Hat sich da was geändert?

event.path liefert nur noch `undefined`

**Comment by Zefau** [comment]
Wo denn genau?

**Reply by mcuiobroker** [reply]
Anscheinend bei allen Event s.
addEventlistener ...click usw.

**Comment by mcuiobroker** [comment]
Beispiel
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/scripts-3.1.x-pro/statelist/httpdropdown#scripts-httpdropdown.js

**Comment by Zefau** [comment]
Habe daran eigentlich nichts geändert. Auf die Standard-Event-Listener habe ich auch gar keinen Einfluss würde ich behaupten. Wüsste zumindest nicht, was ich gemacht haben könnte.

Sonst zeig / schick mir mal den Code, dann schaue ich da drüber. Wobei, wenn da mal funktioniert hat und jetzt nicht.. hm.. :-\

**Reply by mcuiobroker** [reply]
Code ist im Link oben drüber. Ich installiere mal die v3.1.2 und schau mal ob ich da noch den event.path bekomme.
-> kein Unterschied. Dann muss es mit einem Windows Update zu tun haben.

Dabei ist aufgefallen. Geht man auf v3.1.2 zurück zerhaut es wieder die Widgets.
![image](https://user-images.githubusercontent.com/84818934/215294941-de0e3a49-58bb-4dc9-858f-9c96ca804ac9.png)

**Comment by mcuiobroker** [comment, marked as answer]
Habe jetzt bei mozilla gefunden.
https://developer.mozilla.org/en-US/docs/Web/API/Event/composedPath
Wurde ersetzt?
Also früher war das event.path.
Prüfe, ob damit die Programme wieder laufen.

![image](https://user-images.githubusercontent.com/84818934/215296746-b074f222-9343-4bb5-b5f4-7701bea46ea5.png)
https://github.com/Leaflet/Leaflet/issues/8226
